src/cloudant/__init__.py

```diff
@@ -62,6 +63,56 @@ def cloudant(user, passwd, **kwargs):
     cloudant_session.disconnect()
 
 @contextlib.contextmanager
+def cloudant_bluemix(service_name, **kwargs):
```

Comment:
Thoughts around using `service_name=None` and defaulting to use the only service found or erroring if there are more than 1 for a default case?
It would avoid the need to transcribe service names into app code for probably the most common use case.